user_guide_src/source/extending/composer_packages.rst

```diff
@@ -90,6 +90,12 @@ be ``YourVendor``. So you would write like the following::
 
 This setting instructs Composer to autoload the source code for your package.
 
+Choosing License
+================
+
+If you are not familiar with open source licenses, see https://choosealicense.com/.
```

Comment:
Is the link clickable? I think there's a format for links.

Reply:
It is clickable.